modules/howtos/pages/managing-connections.adoc

```diff
@@ -147,11 +148,30 @@ NOTE: Any TLS certificates must be set up at the point where the connections are
 [#ssl]
 == Secure Connections
 
-Couchbase Server Enterprise Edition supports full encryption of client-side traffic using Transport Layer Security (TLS).
+Couchbase Server Enterprise Edition and Couchbase Capella support full encryption of client-side traffic using Transport Layer Security (TLS).
 That includes key-value type operations, queries, and configuration communication.
-Make sure you have the Enterprise Edition of Couchbase Server before proceeding with configuring encryption on the client side.
+Make sure you have the Enterprise Edition of Couchbase Server, or a Couchbase Capella account, before proceeding with configuring encryption on the client side.
+
+[{tabs}]
+====
+Couchbase Capella::
++
+--
+The Java SDK includes Capella’s standard root certificate by default, which means you don’t need any additional configuration to enable TLS.
+Simply use `couchbases://` in your connection string, or `SecurityConfig.enableTls(true)` in your `SecurityConfig` settings.
+--
+
+Couchbase Server::
++
+--
+As of SDK 3.4, if your cluster's root certificate is signed by a well known CA (Certificate Authority) and is included in the JVM's trust store, it is no longer necessary to configure the certificate in the `securityConfig` settings.
 
-To configure encryption for the Java SDK:
+[source,java]
+----
+include::example$ManagingConnections.java[tag=tls-no-trust,indent=0]
+----
+
+However, you can still provide a certificate explicitly if necessary:
```

Comment:
@dnault from what I understand, well known CAs should be supported by default now, without any changes in the `securityConfig` settings.

However, I am unclear if this applies to Couchbase Server only, or Capella as well?